pom.xml

```diff
@@ -59,12 +59,12 @@
   <ciManagement />
   <distributionManagement>
     <snapshotRepository>
-      <id>cloudbees-nexus-snapshots</id>
+      <id>sonatype-nexus-snapshots</id>
```

Comment:
Switching to the repository ID I was already using elsewhere, to avoid the need to duplicate credentials section in my `~/.m2/settings.xml`.